components/Dialogs/src/MDCAlertController.m

```diff
@@ -205,10 +179,7 @@ - (BOOL)mdc_adjustsFontForContentSizeCategory {
 - (void)mdc_setAdjustsFontForContentSizeCategory:(BOOL)adjusts {
   _mdc_adjustsFontForContentSizeCategory = adjusts;
```

Comment:
remove backing store

Reply:
Unfortunately that won't work if alertView hasn't been instantiated yet.